bolt/include/bolt/Core/MCPlusBuilder.h

```diff
@@ -1122,6 +1143,51 @@ class MCPlusBuilder {
   /// Return true if the instruction is a tail call.
   bool isTailCall(const MCInst &Inst) const;
 
+  /// Stores NegateRAState annotation on Inst.
```

Comment:
nit: could use `\p` to indicate parameter (same for the new functions below this)

```suggestion
  /// Stores NegateRAState annotation on \p Inst.
```